DuckDuckGo/BrokenSiteInfo.swift

```diff
@@ -24,6 +24,11 @@ public struct BrokenSiteInfo {
 
     static let allowedQueryReservedCharacters =  CharacterSet(charactersIn: ",")
 
+    enum ProtectionsState: String {
+        case enabled = "1"
+        case disabled = "0"
+    }
```

Comment:
1. Not sure what is the expected way of extending the values range here, but maybe a simple bool would suffice?
2. Either with the above change or not I suggest replacing "1"/"0" to "true"/"false" to be consistent with other parameters.

Reply:
Duplicating the response for visibility :)

> @miasma13 as we discussed in MM - this mapping to "1" | "0" was done intentionally to match what 'other' platforms are doing with this parameter - since I'm adding to all 5 :)